src/app/UserLogin.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledLabel = styled.label`
+  display: grid;
+  padding-bottom: 5px;
+  font-size: 0.6em;
+  color: rebeccapurple;
+`;
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-rows: 150px auto;
+  padding: 10px;
+`;
+
+const StyledInput = styled.input`
+  margin: 0 0 5px 0;
```

Comment:
Do you need to set the margin top/right/left to 0? Otherwise you could just use `margin-bottom: 5px`

Reply:
That's true!